docker-compose.base.yml

```diff
@@ -88,6 +88,20 @@ services:
         volumes:
             - ./docker/postgres-init-scripts:/docker-entrypoint-initdb.d
 
+    behavioral_cohorts_db:
+        image: postgres:12-alpine
```

Comment:
Given this is a new DB, any reason why we wouldnt want to use the latest PG? I think 17 is latest